lib/wasi/Cargo.toml

```diff
@@ -58,7 +58,7 @@ pin-project = "1.0.12"
 # Used by the WCGI runner
 hyper = { version = "0.14", features = ["server", "stream"], optional = true }
 wcgi = { version = "0.1.1", optional = true }
-wcgi-host = { version = "0.1.0", optional = true }
+wcgi-host = { version = "0.1.1", optional = true }
```

Comment:
This was a catch-up thing that I intentionally made part of the PR to simplify dependencies